core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1398,7 +1398,7 @@ private void deleteFromEntry(DeleteFileEntry entry) {
         ufsDeleter = new SafeUfsDeleter(mMountTable, delInodes, deleteOptions);
       }
       // Inodes to delete from tree after attempting to delete from UFS
-      List<Inode> inodesToDelete = new LinkedList<>();
+      List<Pair<AlluxioURI, Inode>> inodesToDelete = new LinkedList<>();
```

Comment:
Update the comment about the type?

Reply:
I don't think it's necessary, we don't describe the type for `delInodes` either